packages/components/src/components/radio-group/radio-group.tsx

```diff
@@ -12,6 +12,7 @@ type TRadioGroup = {
     onToggle: (e: ChangeEvent<HTMLInputElement>) => void;
     required?: boolean;
     selected: string;
+    is_left?: boolean;
```

Comment:
why do we need the changes in this file? 
Changing a single word should not affect the design and we cannot change the shared component because of that